magpy/gui/magpy_gui.py

```diff
@@ -2872,6 +2891,47 @@ def onChangeSymbol(self, event):
             self.OnPlot(self.plotstream,self.shownkeylist)
         self.changeStatusbar("Ready")
 
+    def OnFlagClick(self, event):
+        """Mouse event for flagging with double click."""
+        if event.inaxes and event.dblclick:
```

Comment:
same as above, check the simpler case (return) and handle that so the rest of the code is outdented:

```
if not event.inaxes or not event.dblclick:
    return
# other stuff
```